.circleci/config.yml

```diff
@@ -263,6 +263,31 @@ jobs:
           name: Run test
           command: .circleci/unittest/run_test.sh
 
+  unittest_docker_image_build_linux:
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: 2xlarge
+    environment:
+      image_name: torchaudio/unittest
+    steps:
+      - checkout
+      - run:
+          name: Build Docker Image
+          no_output_timeout: "3h"
+          command: |
+            cd .circleci/docker/unittest && docker build . -t ${image_name}:${CIRCLE_WORKFLOW_ID}
+      - run:
+          name: upload docker image
+          no_output_timeout: "1h"
+          command: |
+            set +x
+            export AWS_ACCESS_KEY_ID=${ECR_AWS_ACCESS_KEY}
+            export AWS_SECRET_ACCESS_KEY=${ECR_AWS_SECRET_ACCESS_KEY}
+            eval $(aws ecr get-login --region us-east-1 --no-include-email)
+            set -x
+            docker tag ${image_name}:${CIRCLE_WORKFLOW_ID} 308535385114.dkr.ecr.us-east-1.amazonaws.com/${image_name}:${CIRCLE_WORKFLOW_ID}
+            docker push 308535385114.dkr.ecr.us-east-1.amazonaws.com/${image_name}:${CIRCLE_WORKFLOW_ID}
```

Comment:
I actually think it'd be better to publish these to Docker Hub instead of ECR.

Management is easier there for us, i.e. creating new images, introspection into tags, etc.

WDYT @ezyang?